data/.schema.yaml

```diff
@@ -91,6 +91,20 @@ properties:
       required:
         - measurements
       properties:
+        attributes:
+          type: object
+          properties:
+            sex:
+              description: Sex of the participant at birth.
+              type: string
+              enum:
+                - female
+                - male
+            age:
```

Comment:
Do we also consider months? As there could be some studies focusing on little children (pediatric studies).

Reply:
Good point! We could use something like 0.5 years for six months. It's a bit annoying for 0.08333.... to represent one month, but I think having consistent units would be good.